tools/setup_helpers/extension.py

```diff
@@ -21,6 +21,10 @@
 _ROOT_DIR = _THIS_DIR.parent.parent.resolve()
 
 
+def _get_cxx11_abi():
+    return "-D_GLIBCXX_USE_CXX11_ABI=" + str(int(torch._C._GLIBCXX_USE_CXX11_ABI))
```

Comment:
nit: Perhaps we can use `torch.compiled_with_cxx11_abi` instead of `torch._C._GLIBCXX_USE_CXX11_ABI`.

https://github.com/pytorch/pytorch/search?l=Python&q=_GLIBCXX_USE_CXX11_ABI

Reply:
Done